include/seqan3/alignment/configuration/align_config_output.hpp

```diff
@@ -0,0 +1,54 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \brief Provides configuration for alignment output.
+ * \author Rene Rahn <rene.rahn AT fu-berlin.de>
+ */
+
+#pragma once
+
+#include <seqan3/alignment/configuration/detail.hpp>
+#include <seqan3/alignment/pairwise/alignment_result.hpp>
+#include <seqan3/core/algorithm/pipeable_config_element.hpp>
+#include <seqan3/core/concept/core_language.hpp>
+
+namespace seqan3::align_cfg
+{
+
+/*!\brief Tag representing score output for the alignment algorithms.
+ * \ingroup alignment_configuration
+ */
+struct output_score_tag : public pipeable_config_element<output_score_tag>
```

Comment:
No. In case we have a simple configuration we decided to call the class `_tag` and the instance of it without the tag suffix. This has nothing to do with detail or non-detail code.

Reply:
> In case we have a simple configuration we decided to call the class _tag and the instance of it without the tag suffix

Yes absolutely

>  This has nothing to do with detail or non-detail code.

I still think so, at least it is that way in all the other condifurations:
`seqan3::detail::method_local_tag` for `seqan3::align_cfg::method_local`
`seqan3::detail::vectorised_tag` for `seqan3::align_cfg::vectorised`
`seqan3::detail::output_query_id_tag` for `seqan3::search_cfg::output_query_id`
...

Reply:
No 😱 they should not be in detail. Otherwise, how is the user supposed to call get on it. `seqan3::get<seqan3::align_cfg::method_local>` is not working and if the tag is in detail namespace, then he can't see the user documentation.

Reply:
Haha well somewhere this went wrong then. I always thought that all the special configuration functionality except the `|` was rather for us internally anyway. Will you open an issue for this? I guess it should be in the current release then.

Reply:
> Haha well somewhere this went wrong then.

Yes, by neither reading the user story nor the config design document. 😒